Month1/Week2/Section1/Exercise3/Main3.cpp

```diff
@@ -73,18 +78,21 @@ class FCircle
 	float DiameterCircle;
 public:
 	FCircle()
+	{
+	}
+	FCircle(float RadiusCircle,float DiameterCircle)
```

Comment:
Same comment here about the constructor of the square

Reply:
updated